packages/framework/src/validators/validator.test.ts

```diff
@@ -193,130 +193,87 @@ describe('validators', () => {
         result: {
           success: false,
           errors: {
-            json: [{ message: "must have required property 'numberType'", path: '' }],
+            json: [
+              { message: "must have required property 'numberType'", path: '' },
+              { message: "must have required property 'booleanType'", path: '' },
+            ],
             zod: null, // Zod has no support for `allOf`
           },
         },
       },
       {
         title: 'should successfully validate polymorphic `anyOf` properties',
         schemas: {
-          zod: z.object({
```

Comment:
The following test updates change the test subject from an polymorphic array of items to a polymorphic object, better matching the typical usage of `anyOf` and working around the short-circuit issue in ajv: https://github.com/ajv-validator/ajv/issues/662